evm/Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-plonky2 = { path = "../plonky2" }
+plonky2 = { path = "../plonky2", default-features = false, features = ["rand", "rand_chacha", "timing", "gate_testing"] }
```

Comment:
I don't think we need `rand_chacha` or `gate_testing`, but I can remove them after.